src/methods.jl

```diff
@@ -145,7 +145,7 @@ julia> trim(frame, (:, 2:5))
 ```
 
 # See Also
-[`trimview`](@ref)
+* [`trimview`](@ref)
```

Comment:
I actually think a comma separated list is easier to read and less distracting than a bulleted list.